server/src/controller/CommentsRouter.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const router = express.Router();
+const commentsService = require('../service/CommentsService');
+const { validateNewComment } = require('./CommentsHelper')
+const { authenticateToken } = require('../util/WebToken');
+
+// CREATE
+// New Comment Post
+//will add authorization check
+router.post('/', validationMiddleware, async (req, res) => {
+    //either throws an error or returns the comment it stores
+    try {
+        //add username to body through the jwt tokens username
+        const data = await commentsService.createNewComment(req.body);
+        res.status(201).json({
+            message: 'Created Comment Successfully',
+            ReviewPost: data
+        });
+    } catch(err) {
+        res.status(500).json({
+            message: `Error Trying This Request`,
+            error_message: err.message
+        })
+    }
+})
+
+// READ
+
+// UPDATE
+
+// DELETE
+
+
+// Validation Middleware
+function validationMiddleware(req, res, next) {
```

Comment:
If you wanted to, you could move this function into helper file. See authenticateToken in WebToken.js as an example